app/i18n/ar.js

```diff
@@ -0,0 +1,63 @@
+export default {
+    Header: {
+        subtitle: 'أسلوب جيّد للتّوبيخ بطريقة مرنة ',
+        logout: 'خروج',
+        leave: 'غادر',
+        summaryMode: 'النّمط الملخّص'
+    },
+    LanguagePicker: {
+        header: 'إختيار اللُّغة'
+    },
+    Main: {
+        hint: 'يمكنُكَ دعوة أشخاص إلى هذه الجلسة عن طريق نسخ عنوان هاته الصفحة'
+    },
+    Post: {
+        vote: 'صوت',
+        votes: 'أصوات',
+        deleteButton: 'حذف',
+        noContent: '(هذا الموضوع فارغ)'
+    },
+    PostBoard: {
+        notWellQuestion: 'ما الذي يمكن تحسينه ؟',
+        wellQuestion: 'ما الذي فُعِلَ بنجاح؟',
+        ideasQuestion: 'هل من أيِّ فكرةٍ عبقيِّة ؟'
+    },
+    Clients: {
+        header: 'الضيوف الحاضرين معنا الآن:'
+    },
+    Join: {
+        welcome: 'مرحبا!',
+        standardTab: {
+            header: 'موضوع جديد',
+            text: 'انقر أدناه و إبدء إعادة النظر:',
+            button: 'إنشاء جلسة عمل جديدة'
+        },
+        advancedTab: {
+            header: 'أكثر',
+            input: 'عنوان',
+            button: 'إنشاء جلسة مخصصة'
+        },
+        previousTab: {
+            header: 'سابق',
+            rejoinButton: 'الإنضمام'
+        }
+    },
+    Login: {
+        namePlaceholder: 'من أنت بالضبط؟ أدخل اسمك هنا',
+        buttonLabel: 'لنبدأ'
+    },
+    SummaryBoard: {
+        noPosts: 'لا توجد مشاركات للعرض'
+    },
+    SessionName: {
+        defaultSessionName: 'جلسة جديدة'
+    },
+    Invite: {
+        inviteButton: 'دعوة',
+        dialog: {
+            title: 'دعوة أشخاص إلى الجلسة',
+            text: 'إرسل العنوان بكلّ بساطة'+'لدعوة أشخاص إلى جلسة',
```

Comment:
not sure why the concatenation here. there is a missing space between basatah and lida3wet